webapp-django/crashstats/documentation/jinja2/documentation/supersearch/examples.html

```diff
@@ -10,12 +10,12 @@ <h2 id="list-of-signatures">List of signatures</h2>
 
     <h3>Goal</h3>
     <p>
-      Get the top 20 signatures for {{ product }} {{ active_versions[product][0].version }}.
+      Get the top 20 signatures for {{ product.name }} {{ active_versions[product.name][0].version }}.
```

Comment:
It's interesting to note that this page no longer works if you don't have enough crash report data to cover the default product. I'm not sure what to do about that. While I think this page is somewhat fancy, I think I'd rather it contained hard-coded examples.